index.html

```diff
@@ -205,10 +205,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
 
       </div>
       <div class="form-inputs">
-        <input type="text" placeholder="Full name">
-        <input type="text" placeholder="Email address">
-        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/mlezozpr" method="post">
+        <input name="name" type="text" placeholder="Full name" maxlength="50" required>
+        <input name="email" type="email" placeholder="Email address" maxlength="50" required>
```

Comment:
- [ ] You have set a `maxlength=50` for your inputs but the specification clearly advises `30`. Please make this change